testing/README.md

```diff
@@ -87,6 +88,17 @@ Deno.test("isNotStrictlyEqual", function (): void {
 });
 ```
 
+Using `assertSnapshot()`:
+
+```ts
+import { assertSnapshot } from "https://deno.land/std@$STD_VERSION/testing/asserts.ts";
+
+Deno.test("isSnapshotMatch", function (test): void {
+  const a = {};
+  assertSnapshot(test, a);
+});
```

Comment:
We could use some more data in this example - please add some fields to the object. Also mention the required flag to update the snapshot

Reply:
Added some fields to the example as requested. I also added a new `## Snapshot Testing` section as I felt there was too much that should be documented to put it under the usage example. Hope this is okay 🙂

Reply:
Also added some test data to the map in `asserts_test.ts` 👍

Ready for re-review @bartlomieju @kt3k